views/home.hbs

```diff
@@ -0,0 +1,6 @@
+<title> Search for an Artist</title>
+
+<form id = "form" role = "search">
```

Comment:
if you don't use `action` with the form, then the form redirects to the same page. In this case, it should have probably redirected to `/artist-search`